guides/source/users/promotions/promotion-calculators.html.md

```diff
@@ -0,0 +1,158 @@
+# Promotion calculators
+
+When you create a promotion action, one step is to select the promotion
```

Comment:
Maybe add a short sentence what a calculator does. It's often not clear for non technical people what this thing is for, besides that it 'calculates things'.